data/json/items/gun/50.json

```diff
@@ -109,5 +109,53 @@
     "magazine_well": 1,
     "magazines": [ [ "50", [ "as50mag" ] ] ],
     "flags": [ "NEVER_JAMS" ]
+  },
+  {
+    "id": "dsr50",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "DSR-50",
+    "description": "Bullpup, Bolt-Action anti-material rifle made by DSR-Precision, successor of DSR-1, the bullpup design make this weapon balanced and manageable.",
+    "weight": "10300 g",
+    "volume": "3100 ml",
+    "price": 12500000,
+    "to-hit": -1,
+    "bashing": 12,
+    "material": "steel",
+    "ammo": "50",
+    "range": 50,
+    "ranged_damage": -5,
+    "dispersion": 110,
+    "durability": 8,
+    "reload": 450,
+    "barrel_length": 5,
+    "default_mods": [ "recoil_stock", "rifle_scope", "muzzle_brake", "bipod" ],
+    "magazine_well": 1,
+    "magazine": [ [ "50", [ "dsr50mag" ] ] ],
+    "flags": [ "NEVER_JAMS" ]
+  },
+  {
+    "id": "hecate2",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "PGM Hecate II",
+    "description": "Bolt-Action anti-material rifle made by PGM Precision. standard sniper rifle of French Army, this is the biggest weapon ever made by PGM Precision.",
```

Comment:
```suggestion
    "description": "Bolt-action anti-materiel rifle made by PGM Precision.  Standard sniper rifle of French Army, this is the biggest weapon ever made by PGM Precision.",
```

Reply:
thank you, blame my auto-capital that was turned on during the writing.